polkadot/node/core/approval-voting/tests/approval-voting-regression-bench.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use polkadot_subsystem_bench::{
+	approval::{bench_approvals, prepare_test, ApprovalsOptions},
+	configuration::TestConfiguration,
+	usage::BenchmarkUsage,
+};
+
+const BENCH_COUNT: usize = 3;
+const WARM_UP_COUNT: usize = 20;
+const WARM_UP_PRECISION: f64 = 0.01;
+
+fn main() -> Result<(), String> {
+	let mut messages = vec![];
+
+	messages.extend(approvals_no_shows()?);
+	// messages.extend(approvals_throughput()?);
+	// messages.extend(approvals_throughput_best_case()?);
+
+	if messages.is_empty() {
+		Ok(())
+	} else {
+		eprintln!("{}", messages.join("\n"));
+		Err("Regressions found".to_string())
+	}
+}
+
+fn test_configuration() -> TestConfiguration {
+	let mut config = TestConfiguration::default();
+	config.n_validators = 500;
+	config.n_cores = 100;
+	config.min_pov_size = 1120;
+	config.max_pov_size = 5120;
+	config.peer_bandwidth = 524288000000;
```

Comment:
Is this value correct as per validator specs (500 Mbit/s (= 62.5 MB/s)) ?

Reply:
@alexggh what do you think? I just copied it from your yaml.